src/components/layer-timestamp-card/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./layer-timestamp-card.vue"
```

Comment:
is a linter function properly set? (no new line message?) we prefer eslint of prettier (no trailing comma's and ; after each line needed)